src/families/tezos/DelegateFlow/steps/StepConfirmation.js

```diff
@@ -85,26 +93,35 @@ function StepConfirmation({
             <IconTriangleWarning height={16} width={16} />
           </Box>
           <Box style={{ display: 'block' }} ff="Inter|SemiBold" fontSize={3} horizontal shrink>
-            <Trans i18nKey="send.steps.confirmation.broadcastError" />
+            <Trans i18nKey="delegation.flow.steps.confirmation.broadcastError" />
           </Box>
         </Disclaimer>
       ) : null}
-      <TrackPage category="Send Flow" name="Step Confirmed" />
+      <TrackPage
+        category={undelegating ? 'Undelegation Flow' : 'Delegation Flow'}
+        name="Step Confirmed"
+      />
       <span style={{ color: iconColor }}>
         <Icon size={43} />
       </span>
       <Title>
         {error ? (
           <TranslatedError error={error} />
         ) : optimisticOperation ? (
-          <Trans i18nKey="send.steps.confirmation.success.title" />
+          // TODO: Contextualize message for undelegation
```

Comment:
what do you mean for this todo?

Reply:
I'm waiting for the correct undelegation success text